core/src/Cabana_CommunicationPlan.hpp

```diff
@@ -300,6 +300,23 @@ class CommunicationPlan
         MPI_Comm_dup( comm, &_comm );
     }
 
+    /*!
+      \brief Copy constructor.
+    */
+    CommunicationPlan( const CommunicationPlan &cp )
```

Comment:
You need to copy these guys too
https://github.com/ECP-copa/Cabana/blob/a8554efcfae9168e83868a173de58faecc7f8fc1/core/src/Cabana_CommunicationPlan.hpp#L821-L827

Reply:
Done